Lib/email/_header_value_parser.py

```diff
@@ -1967,6 +1979,105 @@ def get_address_list(value):
             value = value[1:]
     return address_list, value
 
+
+def get_no_fold_literal(value):
+    """
```

Comment:
What about have the docstring on one line?

```
def get_no_fold_literal(value):
    """no-fold-literal = "[" *dtext "]""""
````

?

Reply:
Do you think this would work?
```
"""no-fold-literal = "[" *dtext "]

"""
```
Again, just for consistency. Other methods in this module use the same format.